master/buildbot/status/buildstep.py

```diff
@@ -248,10 +248,11 @@ def addLog(self, name):
         d.addCallback(self.logFinished)
         return log
 
-    def addHTMLLog(self, name, html):
+    def addHTMLLog(self, name, html, type):
```

Comment:
Same here: we need to make type optional or set it different so we don't break buildbot's api